routes/user/user.controller.js

```diff
@@ -81,48 +91,240 @@ module.exports.deletetable = (req, res) => {
-  if (err) {
-    throw err;
-  }
-  res.json({ success: true, rows: rows });
-});
+
+    date = new Date()
+    userID = parseInt(10000000000000 + date.getTime());
+    //console.log(userID)
+    sql = `INSERT INTO Mentors VALUES (?, ? , ?, ?, ?, ?, ?, ?, ?, ?, ?) `
+    console.log(sql);
+    db.all(sql, [userID, user.first_name, user.last_name, user.email_address,
+      user.biography, user.zipcode, user.date_of_birth, user.profession,
+      user.skills, user.profile_pic_URL, user.hobbies], (err, rows) => {
+        if (err) {
+          throw err;
+        }
+      });
+
+    //add user ID to skills table 
+    skill = user.skills;
+    sql3 = `SELECT users FROM Skills WHERE skills = ?`;
+    db.all(sql3, [skill], (err, rows3) => {
+      if (err) {
+        throw err;
+      }
+      if (rows3.length == 0) {
+        sql4 = `INSERT INTO Skills VALUES (?,CAST(? AS int))`
+        db.all(sql4, [skill, parseInt(userID)], (err, rows4) => {
+          if (err) {
+            throw err;
+          }
+          console.log("insert users into new skill")
+        });
+      } else {
+        users = rows3[0]['users']
+        users = addToString(users, userID);
+
+        sql5 = `UPDATE Skills SET users = ? WHERE skills = ?`
+        db.all(sql5, [users, skill], (err, rows5) => {
+          if (err) {
+            throw err;
+          }
+          console.log("add user to existed users list")
+        });
+      }
+    });
+
+    //add user ID to profession table 
+    profession = user.profession;
+    sql3 = `SELECT users FROM Profession WHERE profession = ?`;
+    db.all(sql3, [profession], (err, rows3) => {
+      if (err) {
+        throw err;
+      }
+      //this means profession doesnt exist 
+      if (rows3.length == 0) {
+        sql4 = `INSERT INTO Profession VALUES (?,CAST(? AS int))`
+        db.all(sql4, [profession, userID], (err, rows4) => {
+          if (err) {
+            throw err;
+          }
+          console.log("new profession added, user attached");
+        });
+      }
+      //profession exists, append the user to list  
+      else {
+        users = rows3[0]['users']
+        users = addToString(users, userID)
+        //add the users to the profession 
+        sql5 = `UPDATE Profession SET users = ? WHERE profession = ?`
+        db.all(sql5, [users, profession], (err, rows5) => {
+          if (err) {
+            throw err;
+          }
+          console.log("Appended a user to an existed users list")
+        });
+      }
+    });
+
+    res.json({ success: true, rows: "" });
+  });
 }
 //create new mentee
 module.exports.postMentee = (req, res) => {
-  const fields = ['first_name', 'last_name', 'email_address' ,'biography','zipcode',
-  'date_of_birth','skills','hobbies'];
+  const fields = ['first_name', 'last_name', 'email_address', 'biography', 'zipcode',
+    'date_of_birth', 'skills', 'profession', 'profile_pic_URL', 'hobbies'];
+  const user = {};
+  fields.forEach(field => {
```

Comment:
Switch `forEach` with `some` and create variable `missingFields` similar to the following format:
https://github.com/RiseCareerAcademy/Rise/blob/6f285da2a2b7a00b01f4925c39e960ee1f9c2fcc/routes/user/user.controller.js#L94